en/middleware/micro_dds_edits.md

```diff
@@ -0,0 +1,377 @@
+# MicroDDS
```

Comment:
I'm guessing you have micro_dds_edits and micro_dds so you can track changes. That's what github does for you. From a reviewer point of view, having both is not using the tools. I'll delete.

Reply:
Yes, sure

Reply:
Deleted now.